notebooks/Test_Lens_class.ipynb

```diff
@@ -0,0 +1,3718 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.        deflector_type="all-galaxies",
</pre><p>here there are twice the deflector_type set, which also causes and error in the pre-commit CI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>